### PR TITLE
Update sx9.json

### PR DESCRIPTION
**Pull request by satr14washere**

### CODE REVIEW

File: domains/sx9.json [hunk 1]
```diff
@@ -4,13 +4,9 @@
   "owner": { 
     "username": "SX-9", 
     "twitter": "SX_Discord",
-    "email": "",
-    "emails": { 
-      "gmail": "2030.agent.discord@gmail.com", 
-      "outlook": "sx-91@outlook.com" 
-    } 
-  }, 
+    "email": "sx-91@outlook.com",
```
Comment (MaskDuck) [inline domains/sx9.json:7]: trailing comma

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sx9.json (hunk 1 above)
Comment (SX-9) [reply domains/sx9.json:7]: Ok

**Review by MaskDuck — APPROVED** [review]
gud

**Comment by SX-9** [comment]
Oh nvm cancel it

**Comment by SX-9** [comment]
Tge hosting is not working

File: domains/sx9.json [hunk 2]
```diff
@@ -4,13 +4,9 @@
...
     "username": "SX-9", 
     "twitter": "SX_Discord",
-    "email": "",
-    "emails": { 
-      "gmail": "2030.agent.discord@gmail.com", 
-      "outlook": "sx-91@outlook.com" 
-    } 
-  }, 
+    "email": "sx-91@outlook.com"
+  }
```
Comment (MaskDuck) [inline domains/sx9.json:8]: where is the comma

**Comment by SX-9** [comment]
*the

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by SX-9** [comment]
Wait

**Comment by SX-9** [comment]
Now merge

**Review by MaskDuck — APPROVED** [review]
no

**Comment by SX-9** [comment]
> no

-_-

**Pull request merged**